Specs/DateTime/English/DatePeriodExtractor.json

```diff
@@ -2649,4 +2649,76 @@
     }
   ]
 }
+,
+{
+  "Input": "In the 1970s",
```

Comment:
Does "the 2000s" work too? Can we add a test case for it?

Reply:
Add the test case. It works.